main.py

```diff
@@ -0,0 +1,26 @@
+negative_word_list = ["bad", "ugly", "terrible", "awful", "stupid", "mad", "angry", "sad"]
+
+
+def sentiment_analysis(text_excerpt):
+    # replace all punctuations with space and convert to list of words
+    text_excerpt = text_excerpt.replace(" ", " ").replace(",", " ").replace(".", " ").replace("!", " ").lower().split()
```

Comment:
import re and use re.sub() instead. That's a more cleaner way than calling replace() multiple times